### PR TITLE
Mule Agent Plugin v2.3.0 release notes

### PR DESCRIPTION
**Pull request by aljrubior**

### CODE REVIEW

File: modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+_April 22, 2019_
+
+These are the new features and enhancements, known limitations, issues, and fixes in Anypoint Runtime Manager agent, version 2.3.0.
+
+== New Features and Enhancements
+
+This release contains the following new features or enhancements:
+
+* Secrets (`clientId`, `clientSecret`, and `Proxy password`) can now be stored, encrypted, in the `wrapper.conf` file when an encryption key is provided at setup time.
```
Comment (mcocaro) [inline modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: lets rename to API Manager Secrets ...

Comment (mcocaro) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: where's the Doc link to how this can be done? @balbifm

Comment (balbifm) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: It is not merged yet, but I guess that the link will be kept: api-manager/2.x/api-gateway-encryption-mule4.adoc

Comment (valkyrie69) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: @balbifm Federico, that link doesn't work. Is that doc published? We can't link to that unless it's already published.

Comment (mcocaro) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: As per talking to Fernando, Doc will be the following and will go out as soon as its reviewed: ```xref:api-manager::api-gateway-encryption-mule4.adoc[Gateway Startup Encryption in Mule 4]```

Comment (mcocaro) [reply modules/ROOT/pages/runtime-manager-agent/runtime-manager-agent-2.3.0-release-notes.adoc]: need to include a disclaimer that the encryption is only going to work with Mule 4.2.0+ or 3.9.3+

**Pull request merged**